json-endpoints/class.wpcom-json-api-get-site-v1-3-endpoint.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+new WPCOM_JSON_API_GET_Site_V1_3_Endpoint(
+	array(
+		'description'        => 'Get information about a site.',
+		'group'              => 'sites',
+		'stat'               => 'sites:X',
+		'allowed_if_flagged' => true,
+		'method'             => 'GET',
+		'min_version'        => '1.3',
+		'max_version'        => '1.3',
+		'path'               => '/sites/%s',
+		'path_labels'        => array(
+			'$site' => '(int|string) Site ID or domain',
+		),
+		'query_parameters'   => array(
+			'context' => false,
+		),
+		'response_format'    => WPCOM_JSON_API_GET_Site_V1_3_Endpoint::$site_format,
+		'example_request'    => 'https://public-api.wordpress.com/rest/v1.3/sites/en.blog.wordpress.com/',
+	)
+);
+
+class WPCOM_JSON_API_GET_Site_V1_3_Endpoint extends WPCOM_JSON_API_GET_Site_V1_2_Endpoint {
+	protected function render_option_keys( &$options_response_keys ) {
```

Comment:
Why is this needed by reference?

Reply:
I saw it is also passed by reference on the parent so I thought we need the same interface since we're extending the method?

https://github.com/Automattic/jetpack/blob/1531b954ebd392ca2693e28549b2333fd39b6a47/json-endpoints/class.wpcom-json-api-get-site-endpoint.php#L378

Reply:
You're right, ignore me – it also throws a warning (saying as much) if you try to not use it 🙃